### ISSUE TITLE
Enable mulit-load csv files

### ISSUE THREAD

**Issue by michael-okeefe** [bigladder/erin#40]
# Pre-Condition

Ensure that grouping all CSV data into a single file vs loading these all individually will actually provide significant time savings. See #34 

# Problem

Currently, ERIN loads multiple (tens or hundreds) of individual CSV files to get loads for individual buildings electrical and thermal usage. This is a performance bottleneck and requires the management of a large number of individual files.

# Proposed Solution

Enable ERIN to load a multi-part-load csv file. That is, enable ERIN to load a single CSV file that holds all of the other loads as well. This file should have a header as follows (NOTE: the header formatting is an accidental issue with using github tables -- the header is just another row in the CSV):

| tag-1 | size-1 | tag-2 | size-2 |
|--------|--------|--------|--------|
| hours | kW | hours | kW |
| 0 | 123.4 | 0 | 123.4 |
| 1 | 123.4 | 2 | 123.4 |
| 2 | 123.4 | 4 | 123.4 | 

Note that we list the tag (the item used in the input TOML file after `loads.<this-is-the-tag>`) and also the size to facilitate
ease of parsing the data into an array (for example, we can reserve the correct vector size ahead of time).

The proposed solution would be two parts:

- create a new ERIN subcommand called `erin pack-loads <input.toml> --outcsv <default:packed-loads.csv>`
- enable a new csv processing command called `multi_part_csv = "csv-file-name.csv"`

This issue should focus on the first bullet above. A separate issue will be created for the second item.

A brief overview of the procedure is as follows:

- load an input toml file exactly like the `erin run` command up to the point of creating the Simulation object
- export all of the loads in the Simulation object (see [this line](https://github.com/bigladder/erin/blob/f2fcd465d58a6b5350499d0550155dd93adba1df/include/erin_next/erin_next_simulation.h#L23)) to the outcsv

NOTE: the LoadDict struct is defined here:
https://github.com/bigladder/erin/blob/f2fcd465d58a6b5350499d0550155dd93adba1df/include/erin_next/erin_next.h#L73

IMPORTANT: work on this task is contingent on determining that consolidating all loads into a single CSV file will actually make a difference in terms of read time! It is important that we benchmark the csv reading using the Ft Illinois case or by doing a similar exercise (e.g., time reading all the Fort Illinois csvs read one-by-one vs manually consolidating a few and extrapolating the difference -- the columns in a single CSV can just be cut/paste to simulate a large multi-part file).

REFERENCE: #34
